pom.xml

```diff
@@ -43,7 +43,7 @@
     <modules>
         <module>common</module>
         <module>test-support</module>
-        <module>kafka-connect-neo4j</module>
+<!--        <module>kafka-connect-neo4j</module>-->
```

Comment:
should we remove this and even the source folder, maybe as a separate PR?

Reply:
Yes I commented this to open a discussion about it. As we don't need it anymore we can get rid of it in this branch